frontend/src/metabase/lib/markdown/getLeadingText.ts

```diff
@@ -0,0 +1,20 @@
+import { parseMarkdown } from "./parseMarkdown";
+import type { Node } from "./types";
+
+export const getLeadingText = (value: string): string => {
```

Comment:
We don't create a separate module per (utility) function. In this case it would be a single `markdown.ts` module with all the functions in the folder. If you feel doing otherwise I'd recommend starting an RRC and getting approvals first before changing the approach.

Reply:
a6ffcfff32a790e45d017861fbfb4bab7e47fe6c